scripts/linters/js_ts_linter_test.py

```diff
@@ -105,12 +105,13 @@ def mock_print(*args):
         self.print_swap = self.swap(python_utils, 'PRINT', mock_print)
 
     def test_validate_and_parse_js_and_ts_files_with_exception(self):
-        def mock_parse_script(unused_file_content):
-            raise Exception()
+        def mock_parse_script(unused_file_content, comment):  # pylint: disable=unused-argument
+            raise Exception('Dummy Exception')
```

Comment:
Make it specific kinda: exception raised from parse_script(

[Ditto in other similar places]